TradeItIosTicketSDK2/TradeItOrder.swift

```diff
@@ -79,19 +77,27 @@ public typealias TradeItPlaceOrderHandlers = (
 
     @objc public func estimatedChange() -> NSDecimalNumber? {
         var optionalPrice: NSDecimalNumber?
-        let type = self.type
-        switch type {
+
+        switch self.type {
         case .market: optionalPrice = quoteLastPrice
         case .limit: optionalPrice = limitPrice
         case .stopLimit: optionalPrice = limitPrice
         case .stopMarket: optionalPrice = stopPrice
         case .unknown: optionalPrice = 0.0
```

Comment:
When the server passes back a price type we don't know about - this is what I was ranting about last week because we're straddling between a data driven design but still have concrete classes for the same concepts.